EECS3311-PD1/src/main/java/org/example/IRequest.java

```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+public interface IRequest {
+
+    public String getCountry();
+
+    public String getIndicatorID();

```

Comment:
nit: `indicatorId` in AbstractFetcher and `indicatorID` in IRequest
Might be better to have the same naming across the codebase :)